src/test/osd/osd-scrub-repair.sh

```diff
@@ -75,8 +75,13 @@ function corrupt_and_repair_two() {
     #
     # 1) remove the corresponding file from the OSDs
     #
-    objectstore_tool $dir $first SOMETHING remove || return 1
-    objectstore_tool $dir $second SOMETHING remove || return 1
+    pids=""
+    run_in_background pids objectstore_tool $dir $first SOMETHING remove
```

Comment:
should something fail here, it will be very difficult to sort out what comes from the first process and what comes from the second process. I suggest run_in_background redirects stdout/stderr and print them in order, when the process terminates. Since the pids are collected in order, the output will also be displayed in order, which will help a lot to debug.